src/base/server_type.cpp

```diff
@@ -125,7 +126,21 @@ int init_server_type (const char *db_name)
 #endif
   if (g_server_type == SERVER_TYPE_TRANSACTION)
     {
-      er_code = ats_Gl.boot (db_name);
+      assert (ts_Gl == nullptr);
+
+      if (is_active_transaction_server())
+	{
+	  ts_Gl.reset (new active_tran_server());
+	}
+      else if (is_passive_transaction_server())
+	{
+	  ts_Gl.reset (new passive_tran_server());
+	}
+      else
+	{
+	  assert (false);
+	}
+      er_code = ts_Gl->boot (db_name);
```

Comment:
Space after function name.
```suggestion
      if (is_active_transaction_server ())
	{
	  ts_Gl.reset (new active_tran_server ());
	}
      else if (is_passive_transaction_server ())
	{
	  ts_Gl.reset (new passive_tran_server ());
	}
      else
	{
	  assert (false);
	}
      er_code = ts_Gl->boot (db_name);
```